### DISCUSSION TITLE
Setting up Gmail as SMTP host

### DISCUSSION THREAD

**Discussion by iain-neirfeno** [requarks/wiki#6794] [Error / Bug Report]
Hi,
I am trying to set GMail as my SMTP host, I am using Google Workspace for this. I am also using SMTP with 2 other services via Google, so I know it is nothing on the Google end.

I have multiple issues:
1. There's no guidance on configuring for Google - being that it is a common use-case, it would be nice for this to be documented.
2. Sending test email does not work, it eventually returns Connection Timeout on the graphQL call.
3. There's no logs - The server does not log anything to suggest that this request was received, nor any other requests - regardless of log level.
4. No combination of toggle switches or ports seem to work to send email - nothing to suggest it is being tried, just a timeout.

Version:
2.5.300

**Comment by mboyden** [comment]
I found this while searching for answers myself. I got it working in 2.5.213 using Gmail. Here is what I can share:

- You have to creat/use an App Password, which means that 2FA must be enabled
  - Once 2FA is turned on, in Security & Sign-in > 2-Step Verification > App Passwords you can create a special PW for just this app (name can be anything). They don't want you to save it, but I do JIC. That's up to you as they are easy to re-create - 1/SaaS app.
- Host: smtp.gmail.com
- Port: 587
- Client Identifying Hostname: I left it blank even though we have a Workspace
- TLS: off
- Verify SSL: off
- Username:/Password: full email/password

Hope that helps. YMMV. Good luck.
